### ISSUE TITLE
News Pagination

### ISSUE THREAD

**Issue by q23** [slackero/phpwcms#155]
dont get news pagination to run with day, month or year. only by limitation is possible yet.
![unbenannt](https://cloud.githubusercontent.com/assets/2988261/11786802/df3cb35a-a288-11e5-93e9-9827bf244608.jpg)

**Comment by slackero** [comment]
Yep, cannot remember why this is there. It has a huge logical problem: where to start, to end. Maybe you have a concept?

**Comment by q23** [comment]
not yet, maybe later in projects we will come back to this

**Comment by q23** [comment]
during creation of new cp news the default setting of dropdown for pagination is 'month'. after saving the new cp its switched to 'entry per page'
